cmd/sourced-imports/main.go

```diff
@@ -10,9 +10,10 @@ import (
 )
 
 var (
-	fRoot    = flag.String("root", ".", "root directory with the analyzed project")
-	fRel     = flag.String("rel", "", "a directory relative to the root to analyze (recursively)")
-	fNum     = flag.Int("n", 0, "max allowed concurrency (0 means use the number of CPUs)")
+	fRoot = flag.String("root", ".", "root directory with the analyzed project")
+	fRel  = flag.String("rel", "", "a directory relative to the root to analyze (recursively)")
+	fNum  = flag.Int("n", 0, "max allowed concurrency (0 means use the number of CPUs)")
```

Comment:
picky one - I think it's not concurrency (there are no multiple concurrent tasks). We just parallelize for loop, so maybe it should be max parallelizm - wdyt?